### DISCUSSION TITLE
Org-wide Discussions?

### DISCUSSION THREAD

**Discussion by dcamron** [ProjectPythia/projectpythia.github.io#260] [Ideas]
It seems to me we have a bit of a discussions organization problem. We have discussions enabled for
* This repo (projectpythia.github.io) - where we of course want direction on the site, its layout, etc.
* Foundations - where we of course want discussions on Foundations content and its organization and direction

However, we also have a lot of discussions about more general projects or endeavors: outreach and collaborations, internships, new responsibilities, project direction, etc. Discussions in these topics seem to fall here on this repo, or randomly between the two, and I find myself having to check both for many of our discussion topics. GitHub now supports [Org-wide Discussions](https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization) (and Team Discussions, if we needed those.) Would this be a good place for more public and more project-wide discussions? You can see one of these newly in use at [Fatiando a Terra](https://github.com/orgs/fatiando/discussions) for context.

**Comment by clyne** [comment]
Agreed. It's a little unclear how to communicate with us. I think a lot of projects that span multiple repos struggle with this. What I think is particularly helpful about the Fatiando site is this Contact Us [page](https://www.fatiando.org/contact/).

**Comment by brian-rose** [comment]
Just noting here that we decided to use the Pangeo discourse as our primary discussion forum:
https://discourse.pangeo.io/c/education/project-pythia/

**Reply by r-ford** [reply]
I think one of the reasons we decided to use Pangeo's Discourse is that we have 3 orgs, so an org-wide Discussions wouldn't cover everything. If we consolidate, could an org-wide Discussions be useful?

**Reply by clyne** [reply]
You raise a good point, @r-ford. I'm not sure how we arrived at the decision to use Pangeo discourse as our primary forum, but if that  still makes the most sense in a single-pythia-organization-world we probably need to do more to communicate that to the Pythia community. For example, our Join Us [link](https://projectpythia.org/#join-us) points to the Pythia discussion forum, not Pangeo.

**Reply by brian-rose** [reply]
The decision to migrate to the Pangeo discourse was both to eliminate redundant discussion boards as well as to be more effective at entraining people from the broader pangeo community into our discussions. 

I think it would be an acceptable compromise to have one single org-wide GitHub discussion board in addition to the pangeo forum. Especially if we can migrate all the existing discussion threads from our various repo-specific discussion boards onto it. (I'm not sure if that's possible)

The broad goal is to make our stuff and our people easily discoverable. Having a GitHub-based discussion board could potentially help reach a different audience than the pangeo discourse. At the slight expense of continuing to have some fractionation of discussion channels. 

I'm fine with that. The main thing I think we should avoid is any further use of repo-specific discussion boards. 

That said, we do have to choose where to direct people as a "primary" point of contact. 

I suggest we update our materials now to point people to the pangeo discourse since it is already established. 

We can then explore the possibility of merging all our existing GitHub discussion boards into one single org-level discussion. If we like that, we can further update our contact pages to also point people to that resource in addition to the pangeo discourse. 

My $0.02

**Reply by clyne** [reply]
I don't really have a strong opinion on having our own (Pythia hosted) discussion forum vs using Pangeo's. What I think is important is that we settle on one - it sounds like Pangeo was the consensus - clearly direct people to it, and if possible remove/disable the others. It would be nice to migrate existing threads, but I'm not sure that is necessary. Maybe just cherry pick threads that seem to be still relevant and cut-n-paste if necessary.
